e2e/src/pages/profile.po.ts

```diff
@@ -0,0 +1,82 @@
+import { $, $$ } from 'protractor';
+import { BaseComponent } from './base.component';
+
+export class ProfilePo extends BaseComponent {
+  readonly url = '/u/';
+
+  private readonly btnEdit = $('.btn-edit-trigger .Button-icon');
+  private readonly btnViewProfile = $('.btn-edit-trigger .fl-icon-user');
+  private readonly iconEditProfHeadline = $('.profile-user-byline .edit-widget-trigger');
+  private readonly iconEditProfSummary = $('#profile-about-description-wrapper .edit-widget-trigger');
+  private readonly iconEditHourRate = $('.profile-hourly-rate .edit-widget-trigger');
+  private readonly inputProfHeadline = $('.edit-widget-input');
+  private readonly inputProfSummary = $('.profile-about-description-input');
+  private readonly inputHourRate = $('.hourly-rate-input');
+  private readonly btnSaveProfHeadline = $('.profile-user-byline [i18n-msg="Save"]');
+  private readonly btnSaveProfSummary = $('#profile-about-description-wrapper [i18n-msg="Save"]');
+  private readonly btnSaveHourRate = $('.profile-hourly-rate [i18n-msg="Save"]');
```

Comment:
plz avoid using attr with i18n prefix in your locators, cause it relates to localization staff and the value of such attr can be changed for another locale